types/node-ipc/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "interface-name": [
```

Comment:
These should all be fixable. For `interface-name`, it does seem unfortunate that tslint disallows names starting in `IPC`, but it shouldn't be necessary to prefix every name with `IPC` anyway since they're already namespaced.